docker-app/qfieldcloud/core/exceptions.py

```diff
@@ -175,3 +175,23 @@ class ReachedMaxOrganizationMembersError(QFieldCloudException):
     code = "organization_has_max_number_of_members"
     message = "Cannot add new organization members, account limit has been reached."
     status_code = status.HTTP_403_FORBIDDEN
+
+
+class AccountInactiveError(QFieldCloudException):
+    """
+    Raised when a permission (action) is denied because the useraccount is inactive.
+    """
+
+    code = "permission_denied_inactive"
+    message = "Permission denied because the useraccount is inactive"
+    status_code = status.HTTP_403_FORBIDDEN
```

Comment:
```suggestion
class InactiveSubscriptionError(QFieldCloudException):
    """
    Raised when a permission (action) is denied because the useraccount subrscription is inactive.
    """

    code = "inactive_subscription"
    message = "Permission denied because the subscription is inactive"
    status_code = status.HTTP_403_FORBIDDEN
```


I think it makes more sense to say that the subscription is inactive, not the account itself.